### ISSUE TITLE
Update babel-preset-poi

### ISSUE THREAD

**Issue by throrin19** [egoist/poi#464]
**Do you want to request a *feature* or report a *bug*?**

A feature

**What is the current behavior?**

For now, babel-preset-poi use version `7.0.0-beta.40`, or, the actual version is `7.0.0-beta.51` and a lot of changes provides with version `7.0.0-beta.48`.

**If this is a feature request, what is the motivation or use case for changing the behavior?**

After beta.47, es6 classes will not works in my code using babel-preset-poi. I made an issue about that in babel issues and, I think the only difference is the versions above babel-core, babel-polyfils and all babel libraries in babel-preset-poi (https://github.com/babel/babel/issues/8063)

**Comment by throrin19** [comment]
Any news @egoist ?

**Comment by egoist** [comment]
I guess It's already updated?

**Comment by throrin19** [comment]
Yep, but now it is a problem with differents plugins configs and deprecations params in Babel

**Comment by egoist** [comment]
which params are deprecated? I thought I've fixed them https://github.com/egoist/poi/commit/319be5f63d61e3348d1ab38387a273f7b032bf1d 🤔
